CMakeLists.txt

```diff
@@ -130,15 +130,19 @@ add_swift_library(Foundation
                     Foundation/CGFloat.swift
                     Foundation/CharacterSet.swift
                     Foundation/Codable.swift
+                    Foundation/Collections+DataProtocol.swift
```

Comment:
If it does, it will be immediately apparent.  The places where things were out of order were due to the compiler requiring the interfaces to be processed earlier.